src/server/snapshot.cc

```diff
@@ -360,4 +364,8 @@ size_t SliceSnapshot::GetTotalChannelCapacity() const {
   return dest_->GetSize();
 }
 
+std::pair<size_t, size_t> SliceSnapshot::GetCurrentSnapshotProgress() const {
+  return {stats_.loop_serialized, stats_.keys_total};
```

Comment:
side_saved + loop_serialized
is the number of entries serialized

Reply:
nice catch!